### PR TITLE
Update navicat-premium.rb

### PR DESCRIPTION
**Pull request by langaobai**
Actual: f46f38799aaa69d830c04f789f917151bdab1eeee617433daaea6d33b5a7bd2d,Already update

**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [ ] `brew cask audit --download {{cask_file}}` is error-free.
- [ ] `brew cask style --fix {{cask_file}}` reports no offenses.
- [ ] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [ ] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [ ] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [ ] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [ ] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/navicat-premium.rb [hunk 1]
```diff
@@ -2,7 +2,7 @@
   version '15.0.17'
 
   language 'zh', 'CN' do
```
Comment (miccal) [inline Casks/navicat-premium.rb:4]: Please change this to:
```
language 'zh-CN' do
```

Comment (miccal) [reply Casks/navicat-premium.rb:4]: Please also update the `sha256` for the `en` version:
```
Error: Checksum for Cask 'navicat-premium' does not match.
Expected: ba0ddffa076c7d01384dedf7383bcc33695d27c584223944e62f8df388403f3d
Actual: abc959616d08b2f0dabe08f40068013bfb716352f725197e2221f92e0e828707
```

Comment (langaobai) [reply Casks/navicat-premium.rb:4]: Yes，Already update sha256,And language

**Comment by ran-dall** [comment]
@langaobai There are no changes in this PR, so I'm closing it.

**Pull request closed without merging**